crest/Assets/Crest/Crest/Shaders/Resources/UpdateFoam.compute

```diff
@@ -41,30 +41,33 @@ void UpdateFoam(uint3 id : SV_DispatchThreadID)
 	const CascadeParams cascadeData = _CrestCascadeData[sliceIndex];
 	const float2 worldPosXZ = UVToWorld(input_uv, sliceIndex, cascadeData);
 	const float3 uv_slice = float3(input_uv, id.z);
+	half foam = 0.0;
```

Comment:
the diff of this file is a little hard to review here which is fine but just checking this is safe to do? the logic below can be a bit delicate but if it looks good ignore this comment

Reply:
It is only whitespace changes sorry. If hide whitespace is enabled it makes it easier:

![1_1](https://user-images.githubusercontent.com/5249806/153779615-5ca55ab6-eeb6-4505-bb3e-f0c52a71cd29.jpg)

Reply:
In case you're on mobile and it isn't available:

![22](https://user-images.githubusercontent.com/5249806/153779793-3032bd15-ebcd-4b53-8e1a-974cab49ae34.jpg)

I might drop the whitespace changes and do separately.

Reply:
Oh forgot about that thanks. Looks good to me, feel free to keep